### PR TITLE
Operator 3.2.4

### PR DESCRIPTION
**Pull request by rjeberhard**
Prepare to release operator 3.2.4 as we have a number of important fixes ready for external and internal customers.

### CODE REVIEW

File: documentation/staging/content/release-notes.md [hunk 1]
```diff
@@ -40,6 +41,18 @@ draft: false
...
 ### Change log
 
+#### Operator 3.2.4
+
+* Added support for the sessionAffinity field for the clusterService ([#2383](https://github.com/oracle/weblogic-kubernetes-operator/pull/2383)).
+* Moved several logging messages to the CONFIG level ([#2387](https://github.com/oracle/weblogic-kubernetes-operator/pull/2387)).
+* Resolved an issue related to scalingAction.sh when there were multiple domains in the same namespace ([#2388](https://github.com/oracle/weblogic-kubernetes-operator/pull/2388)).
+* Updated introspection and other scripts to consistently use ISO-8601 timestamp formatting ([#2386](https://github.com/oracle/weblogic-kubernetes-operator/pull/2386)).
+* Resolved an issue related to monitoring exporter integration and Istio ([#2390](https://github.com/oracle/weblogic-kubernetes-operator/pull/2390)).
+* Additional diagnostics when the Node Manager fails to start ([#2393](https://github.com/oracle/weblogic-kubernetes-operator/pull/2393)).
```
Comment (mriccell) [inline documentation/staging/content/release-notes.md:51]: Is this 2393 related to the liveness probe?  If so I would prefer a more generic description otherwise it would lead to confusion.

File: documentation/staging/content/release-notes.md [hunk 2]
```diff
@@ -40,6 +41,18 @@ draft: false
 
 ### Change log
 
+#### Operator 3.2.4
+
+* Added support for the sessionAffinity field for the clusterService ([#2383](https://github.com/oracle/weblogic-kubernetes-operator/pull/2383)).
+* Moved several logging messages to the CONFIG level ([#2387](https://github.com/oracle/weblogic-kubernetes-operator/pull/2387)).
+* Resolved an issue related to scalingAction.sh when there were multiple domains in the same namespace ([#2388](https://github.com/oracle/weblogic-kubernetes-operator/pull/2388)).
+* Updated introspection and other scripts to consistently use ISO-8601 timestamp formatting ([#2386](https://github.com/oracle/weblogic-kubernetes-operator/pull/2386)).
```
Comment (tbarnes-us) [inline documentation/staging/content/release-notes.md:49]: I thought the operator log's timestamp format also changed in 2386?

Comment (rjeberhard) [reply documentation/staging/content/release-notes.md]: Updated

File: documentation/staging/content/release-notes.md (hunk 1 above)
Comment (rjeberhard) [reply documentation/staging/content/release-notes.md]: Updated

File: documentation/3.2/content/userguide/managing-domains/model-in-image/runtime-updates.md [hunk 3]
```diff
@@ -253,20 +253,27 @@ For online or offline updates:
 
 For offline updates only, there are two additional options:
 
-  - Supply a new image with new or changed model files
-    and use your Domain YAML `spec.image` field to reference the image.
+  - Supply a new image with new or changed model files.
+    - If the files are located in the image specified in the Domain YAML file `spec.image`,
```
Comment (ddsharpe) [inline documentation/3.2/content/userguide/managing-domains/model-in-image/runtime-updates.md:257]: specified in the Domain YAML file -> specified in the domain resource YAML file

File: documentation/3.2/content/userguide/managing-domains/model-in-image/runtime-updates.md [hunk 4]
```diff
@@ -253,20 +253,27 @@ For online or offline updates:
...
+    - If you are using
+      [common mounts]({{< relref "/userguide/managing-domains/model-in-image/common-mounts.md" >}})
+      to supply
+      model files in an image, then change the corresponding `serverPod.commonMounts.image` field
+      value to reference the new image or add a new `serverPod.commonMounts` mount for
+      the new image.
 
   - Change, add, or delete environment variables that are referenced by macros in your model files.
-    Environment variables are specified in the Domain YAML `spec.serverPod.env`
+    Environment variables are specified in the Domain YAML file `spec.serverPod.env`
```
Comment (ddsharpe) [inline documentation/3.2/content/userguide/managing-domains/model-in-image/runtime-updates.md:267]: domain resource YAML

File: documentation/3.2/content/userguide/managing-domains/model-in-image/runtime-updates.md [hunk 5]
```diff
@@ -253,20 +253,27 @@ For online or offline updates:
...
     or `spec.serverPod.adminServer.env` attributes.
 
 {{% notice note %}}
-It is advisable to defer the last two modification options, or similar Domain YAML changes to
+It is advisable to defer the last two modification options, or similar Domain YAML file changes to
```
Comment (ddsharpe) [inline documentation/3.2/content/userguide/managing-domains/model-in-image/runtime-updates.md:271]: domain resource YAML

File: documentation/3.2/content/userguide/managing-domains/model-in-image/runtime-updates.md [hunk 6]
```diff
@@ -289,16 +296,24 @@ After your model updates are prepared, you can instruct the operator to propagat
...
+#### Offline update steps
 
 Use the following steps to initiate an offline configuration update to your model:
 
  1. Ensure your updates are supported by checking [Supported](#supported-updates) and [Unsupported](#unsupported-updates) updates.
  1. Modify, add, or delete your model resources as per [Updating an existing model](#updating-an-existing-model).
  1. Modify your domain resource YAML file:
-    1. If you have updated your image, change `domain.spec.image` accordingly.
+    1. If you have updated your image:
+       - If the files are located in the image specified in the Domain YAML file `spec.image`,
```
Comment (ddsharpe) [inline documentation/3.2/content/userguide/managing-domains/model-in-image/runtime-updates.md:309]: domain resource YAML

**Review by mriccell — APPROVED** [review]

**Review by ddsharpe — APPROVED** [review]
